lib/main_common.pm

```diff
@@ -1433,6 +1429,10 @@ sub load_extra_tests_desktop {
         loadtest "x11/gnucash";
 
     }
+    # https://progress.opensuse.org/issues/37342
+    if (is_sle() && gnomestep_is_applicable()) {
```

Comment:
What about staging? Isn't it really bad for staging if the test takes too long?

Reply:
See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6410#discussion_r241825336